### PR TITLE
Updates to SMS Phone numbers doc

### PR DESCRIPTION
**Pull request by KellieHawks**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Currents<br>Internal Tools<br>Product Partnerships<br>SMS<br>Customer Lifecycle, Identity and Permissions</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Reporting<br>Intelligence<br>User Targeting<br>IAM<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging and Automation<br>Email (Composition and Infrastructure)</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_user_guide/message_building_by_channel/sms/phone_numbers/user_phone_numbers.md [hunk 1]
```diff
@@ -48,20 +48,24 @@ For a customer to receive an SMS message, they must have a valid phone number an
...
 When a phone number is deemed unreachable or invalid, Braze will mark the user's phone number as invalid and will not attempt to send further communications to that phone number. An invalid phone number is marked in the **Engagement Tab** of a user profile.
 
-A phone number is considered invalid in the following cases:
+![][picture2]{: style="max-width:50%;border: 0;"}
 
-- The phone number supplied is not a valid phone number.
-- The phone number is incorrectly formatted.
+A phone number is considered invalid for the following reasons:
+- **Provider Error**: a permanent error was received from the SMS provider. This indicates that phone number supplied is incorrectly formatted or permanently unable to receive SMS messages.
+- **Deactivated**: the phone number has been deactivated due to a mobile subscriber terminating their service is releasing their number from their carrier.
```
Comment (jooshbzm) [inline _docs/_user_guide/message_building_by_channel/sms/phone_numbers/user_phone_numbers.md:55]:
```suggestion
- **Deactivated**: the phone number has been deactivated due to a mobile subscriber terminating their service and releasing their number from their carrier.
```

File: _docs/_user_guide/message_building_by_channel/sms/phone_numbers/user_phone_numbers.md [hunk 2]
```diff
@@ -48,20 +48,24 @@ For a customer to receive an SMS message, they must have a valid phone number an
...
+
+#### Deactivated phone numbers
+When a mobile subscriber terminates or releases their phone number, their phone number becomes deactivated and will eventually get recycled and assigned to new users. On a daily basis, Braze receives these deactivated numbers and marks them as invalid within the Braze platform. This ensures that phone numbers you believe had opted into your program now belong to a different user who has not consented to receive messages.
 
 These invalid phone numbers can be managed using [SMS endpoints]({{site.baseurl}}/api/endpoints/sms/).
```
Comment (jooshbzm) [inline _docs/_user_guide/message_building_by_channel/sms/phone_numbers/user_phone_numbers.md:60]: @ianabels do we want to make any recommendations around unsubscribing the deactivated users as well as removing?

**Review by jooshbzm — APPROVED** [review]
Just a minor suggestion otherwise LGTM

**Review by jooshbzm — APPROVED** [review]
Just a minor suggestion otherwise LGTM

File: _docs/_user_guide/message_building_by_channel/sms/phone_numbers/user_phone_numbers.md (hunk 2 above)
Comment (ianabels) [reply _docs/_user_guide/message_building_by_channel/sms/phone_numbers/user_phone_numbers.md:60]: I considered it but thought the nuance might make it more confusing than necessary. Let's start with this and we can make it more prescriptive if needed. I'll check with PMM on positioning.

**Pull request merged**